### PR TITLE
chore: migrate to new prow reviewing

### PR DESCRIPTION
**Pull request by wuhuizuo**
- Backgroup: https://internals.tidb.io/t/topic/854
- Ref: https://github.com/ti-community-infra/configs/issues/824

Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### What problem does this PR solve?

Issue Number: ref ti-community-infra/configs#824

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Code changes

- Has the configuration change
- Has HTTP API interfaces changed (Don't forget to [add the declarative for the new API](https://github.com/tikv/pd/blob/master/docs/development.md#updating-api-documentation))
- Has persistent data change

Side effects

- Possible performance regression
- Increased code complexity
- Breaking backward compatibility

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):
- PR to update [`pingcap/tiup`](https://github.com/pingcap/tiup):
- Need to cherry-pick to the release branch

### Release note

```release-note
None.
```

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+approvers:
```
Comment (JmPotato) [inline OWNERS:2]: Why not keep the original term committer?

Comment (wuhuizuo) [reply OWNERS:2]: It's talked in https://internals.tidb.io/t/topic/854.

Comment (wuhuizuo) [reply OWNERS:2]: Current it should not merged before all the other jobs be done, I will add a hold label to block it to merge, but we can give it a `LGTM` and `/merge` when we reach an agreement.

Comment (nolouch) [reply OWNERS:2]: +1, `commiter` is a more common term.

Comment (wuhuizuo) [reply OWNERS:2]: It's a struct field, we could consider it equal our `commiter`.
https://www.kubernetes.dev/docs/guide/owners/#owners

image

**Comment by wuhuizuo** [comment]
Ref: https://github.com/tikv/tikv/pull/14513
close it.

**Pull request closed without merging**